test/extensions/filters/udp/udp_proxy/udp_proxy_filter_test.cc

```diff
@@ -47,30 +52,54 @@ class UdpProxyFilterTest : public testing::Test {
     TestSession(UdpProxyFilterTest& parent,
                 const Network::Address::InstanceConstSharedPtr& upstream_address)
         : parent_(parent), upstream_address_(upstream_address),
-          io_handle_(new Network::MockIoHandle()) {}
+          socket_(new NiceMock<Network::MockSocket>()) {
+      ON_CALL(*socket_, ipVersion()).WillByDefault(Return(upstream_address_->ip()->version()));
+    }
+
+    void expectUpstreamWriteOriginalSrcIp() {
```

Comment:
nits: rename to expectSetIpTransparentSocketOption

Reply:
Will be fixed.